### PR TITLE
Minor fixes

### LINKED ISSUE #511: Don't show participants in user screen
It overflows the user screen and adds no meaningful information to the project / subjects screen.

### LINKED ISSUE #512: User filtering does not work
When filtering on users, all users just disappear

### PR DESCRIPTION
**Pull request by nivemaham**
Fixes issues with user status always displayed in Dutch
Fixes issues with user management filters. Fixes #512 
Fixes issues with sorting #515 
Does not show participants in user-management window. Fixes #511

### CODE REVIEW

**Review by blootsvoets — APPROVED** [review]
LGTM

**Review by blootsvoets — DISMISSED** [review]
LGTM

File: src/main/webapp/app/admin/user-management/user-management.component.html [hunk 1]
```diff
@@ -37,11 +37,11 @@ <h4 jhiTranslate="userManagement.search.title">Filter by</h4>
...
-                <th jhiSortBy="lastModifiedDate"><span jhiTranslate="userManagement.role.project">Project</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="createdDate"><span jhiTranslate="userManagement.createdDate">Created Date</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="lastModifiedBy"><span jhiTranslate="userManagement.lastModifiedBy">Last Modified By</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="lastModifiedDate"><span jhiTranslate="userManagement.lastModifiedDate">Last Modified Date</span> <span class="fa fa-sort"></span></th>
+                <th><span jhiTranslate="userManagement.role.authorityName">Profiles</span></th>
+                <th><span jhiTranslate="userManagement.role.project">Project</span></th>
+                <th><span jhiTranslate="userManagement.createdDate">Created Date</span></th>
+                <th><span jhiTranslate="userManagement.lastModifiedBy">Last Modified By</span></th>
+                <th><span jhiTranslate="userManagement.lastModifiedDate">Last Modified Date</span></th>
```
Comment (blootsvoets) [inline src/main/webapp/app/admin/user-management/user-management.component.html:44]: Sorting could not get working here?

**Review by blootsvoets — COMMENTED** [review]
LGTM, just checking whether sorting could work at the user management level, just excluding the `authorityName` and `project`?

File: src/main/webapp/app/admin/user-management/user-management.component.html (hunk 1 above)
Comment (nivemaham) [reply src/main/webapp/app/admin/user-management/user-management.component.html:44]: No. All of these don't work. Authority and Project don't work due to the select issue I talked about.
Rest of the properties have been moved to the revision entity and I couldn't find an easy way to get the sorting work as we want.

**Pull request merged**